src/main/java/io/phasetwo/service/datastore/representation/UserRolesRepresentation.java

```diff
@@ -0,0 +1,27 @@
+package io.phasetwo.service.datastore.representation;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.List;
+import lombok.AllArgsConstructor;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+
+@Getter
+@Setter
+@ToString
+@EqualsAndHashCode
+@NoArgsConstructor
+@AllArgsConstructor
```

Comment:
Why all these and not just `@Data`?

Reply:
We need both 
@NoArgsConstructor
@AllArgsConstructor
The @Data contains only the @RequiredArgsConstructor constructor.